.github/workflows/test.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: test
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+env:
+  GO111MODULE: on
+  DEBIAN_FRONTEND: noninteractive
+jobs:
+  test:
+    strategy:
+      matrix:
+        go: ["1.14.x"]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - run: |
+        sudo apt-get update
+        sudo apt-get install -y redis-server
```

Comment:
*redis-server* is needed for test by **mackerel-plugin-redis**.

see github.com/soh335/go-test-redisserver